src/main/java/core/jdbc/JdbcTemplate.java

```diff
@@ -7,30 +7,33 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public enum JdbcTemplate {
-    INSTANCE;
+public class JdbcTemplate {
 
-    public void update(PreparedStatementCreator preparedStatementCreator) throws SQLException {
-        try (Connection con = ConnectionManager.getConnection();
-            PreparedStatement pstmt = preparedStatementCreator.createPreparedStatement(con))
-        {
+    private Connection con;
+
+    public JdbcTemplate() {
+        con = ConnectionManager.getConnection();
+    }
```

Comment:
```suggestion
    public JdbcTemplate(Connection con) {
        this.con = con;
    }
```
이렇게 외부에서 커넥션을 주입 받아서 사용할 수 도 있을 것 같아요!!
외부에서 계속 커넥션을 가져와야하는 중복적인 로직이 발생할 수 있지만 이러한 불편함을 해소하기 위해 인프라 로직을 분리하기 위한 미션이 후에 있는데요~
이번 가이드는 참고만 하시면 될 것 같아요 😄